src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -2020,5 +2030,209 @@ public String getVariableNamePattern() {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+		final boolean KEYED_BOSS_BARS_EXIST = Skript.classExists("org.bukkit.boss.KeyedBossBar");
```

Comment:
empty line between type registrations, same for all other types
```suggestion
				.serializer(new EnumSerializer<>(Attribute.class)));

		final boolean KEYED_BOSS_BARS_EXIST = Skript.classExists("org.bukkit.boss.KeyedBossBar");
```